### ISSUE TITLE
Missing user guidance?

### ISSUE THREAD

**Issue by egonw** [WDscholia/scholia#1950]
## Context
On the "author" aspect landing page, there are two textboxes without explanation what they are for:

![image](https://user-images.githubusercontent.com/26721/165936162-4d5b94bf-4f20-4369-a6cd-5fcc0bbeab93.png)

## Question
Shall we add a header or some short guidance what people can type there?

**Comment by fnielsen** [comment]
Yes. First we should fix https://github.com/WDscholia/scholia/issues/1799 A change introduced this error.
